### PR TITLE
438 services menu improvements

### LINKED ISSUE #438: [DOC] - Improvements to Services menu in JLab
### Preliminary Checks

- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/nebari-dev/nebari/discussions

### Summary

Need to add docs around the services menu in JupyterLab. 

### Steps to Resolve this Issue

Should be added to https://www.nebari.dev/docs/tutorials/login-keycloak

### PR DESCRIPTION
**Pull request by pt247**
## Reference Issues or PRs

Fixes #438

## What does this implement/fix?
Updates https://www.nebari.dev/docs/tutorials/login-keycloak/ page. 

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [x] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [ ] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [ ] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

**Comment by pt247** [comment]
@kcpevey I have updated the page. Please compare [updated](https://deploy-preview-466--nebari-docs.netlify.app/docs/tutorials/login-keycloak) and [existing](https://www.nebari.dev/docs/tutorials/login-keycloak/).

File: docs/docs/tutorials/login-with-keycloak.md
Comment (kcpevey) [inline docs/docs/tutorials/login-with-keycloak.md:110]:
```suggestion
```

File: docs/docs/tutorials/login-with-keycloak.md
Comment (kcpevey) [inline docs/docs/tutorials/login-with-keycloak.md:115]:
```suggestion
:::warning
- The starting up sequence can take up to several minutes, depending on the size of the cluster. If the server is not accessible **after 10 minutes**, an error will be shown, and you will be redirected to the Home page. Please check the [troubleshooting docs][troubleshooting] for more information.

- The Event logs may include warnings about `"[Warning] 0/2 nodes are available: 2 node(s) didn't match Pod's node affinity/selector ..." `. This is normal behaviour as cluster sometimes need to expand in order to start the server.
- Warning such as `"[Warning]" Unable to retrieve some image pull secrets ..."` can also be safely ignored.
:::
```

**Review by kcpevey — CHANGES_REQUESTED** [review]
Apologies, I wasn't very clear. I'd like the event log warning message to be included at the top level, not inside the hidden expandable section.

File: docs/docs/tutorials/login-with-keycloak.md
Comment (pt247) [reply docs/docs/tutorials/login-with-keycloak.md]: It makes sense.  Done.

File: docs/static/img/tutorials/nebari-home.png
Comment (viniciusdc) [inline docs/static/img/tutorials/nebari-home.png:1]: Hi @pt247, I will rename those custom apps below (trame ones) to something like `app`. Could you then retake the print?

Comment (pt247) [reply docs/static/img/tutorials/nebari-home.png:1]: Done.

File: docs/docs/explanations/advanced-custom-settings.md
Comment (viniciusdc) [inline docs/docs/explanations/advanced-custom-settings.md:1]: @pt247 I see that you already included the keda config section in here, I would like to merge it after the corresponding code is also merged in nebari. But, to not block this one, can you move this changes to a separate PR?

**Comment by viniciusdc** [comment]
Besides the point above, all LGTM. @kcpevey is there anything else you would like to include as part of this?

**Review by kcpevey — APPROVED** [review]
LGTM. Thanks @pt247 !

**Pull request merged**
